packages/hd-transport/src/types/messages.ts

```
@@ -1749,6 +1749,30 @@ export type EthereumSignTypedDataOneKey = {
   chain_id?: number;
 };
 
+export enum EthereumGnosisSafeTxOperation {
+  CALL = 0,
+  DELEGATE_CALL = 1,
+}
+
+// EthereumGnosisSafeTxRequest
+export type EthereumGnosisSafeTxRequest = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)